enlighten/data/ModelFWHM.py

```diff
@@ -54,6 +54,8 @@
     WP-633-ER-R-50      11.8       11.8      13.5       13.5
 """
 
+DEFAULT = 15
```

Comment:
Now that I read this, it should probably be like 15 for Raman instruments (have an excitation), and 3 for non-Raman, since FWHM is a raw scalar and needs interpreted as either 1/cm or nm. But it's still better than what we had so moving forward.